excludes SecurityAutoConfiguration

timoe opened the pull request:
## Description
Spring Sec. Filters avoid invocations of S3 Mocks API- 

## Related Issue
https://github.com/adobe/S3Mock/issues/59

## Tasks

- [x] I have signed the [CLA](http://adobe.github.io/cla.html).
- [x] I have written tests and verified that they fail without my change.

agudian commented:
Would be awesome if you could add a test for it, so that it doesn't break with future versions of SpringBoot.

timoe commented:
Updated.

> Would be awesome if you could add a test for it, so that it doesn't break with future versions of SpringBoot.

I added `spring-security-oauth2` as a test dep. to s3mock-junit4 as this was exactly the test that fails when referencing `spring-security-oauth2`.
